generators/model/index.js

```diff
@@ -118,6 +119,10 @@ module.exports = SourceSelectingSubGenerator.extend({
       this.props = props;
       var modelFileName = props.modelName + 'Model.xml';
       var contextFileName = props.modelName + '-model-context.xml';
+      if (_.isEmpty(props.targetModule)) {
```

Comment:
In my test, when targetModule is empty, line 126 (props.targetModule.module) fails. So, I thought adding this code will not hurt and also help in coverage